app/controllers/mixins/actions/vm_actions/resize.rb

```diff
@@ -86,7 +98,9 @@ def resize_vm
 
         def resize_form_fields
           assert_privileges("instance_resize")
-          @record = find_record_with_rbac(VmOrTemplate, params[:id])
+
+          @request_id, record_id = params[:id].split(/\s*,\s*/, 2)
```

Comment:
Please send those ids in separate params. Do something like this: 
```ruby
record_id = params[:id]
@request_id = params[:req_is]
```
(I hope I didn't switch them in my example :D )